amethyst_network/src/lib.rs

```diff
@@ -28,22 +29,24 @@ pub use {
     network_socket::NetSocketSystem,
 };
 
-use std::net::{SocketAddr, UdpSocket};
+use std::net::SocketAddr;
 
 use bincode::{deserialize, serialize, ErrorKind};
 use serde::{de::DeserializeOwned, Serialize};
+use laminar::net::UdpSocket;
+use laminar::Packet;
 
 /// Sends an event to the target NetConnection using the provided network Socket.
 /// The socket has to be bound.
-pub fn send_event<T>(event: &NetEvent<T>, target: &SocketAddr, socket: &UdpSocket)
+pub fn send_event<T>(event: &NetEvent<T>, addr: &SocketAddr, socket: &mut UdpSocket)
 where
     T: Serialize,
 {
     let ser = serialize(event);
     match ser {
         Ok(s) => {
             let slice = s.as_slice();
-            match socket.send_to(slice, target) {
+            match socket.send(&Packet::unreliable(*addr, slice.to_vec())) {
```

Comment:
Should the reliability be documented?

Reply:
It is in laminar-tester book. And later on we should make this configurable for the user of amethyst network.

Reply:
Yes, but I meant for the docs on this function. Even if it's going to change, I think documentation would be nice.

Reply:
Oke, let me do that.

Reply:
Yeah, just add a doc that at least links to laminar docs explaining what type of message is sent. thanks.